src/v/kafka/server/tests/create_topics_test.cc

```diff
@@ -104,6 +104,43 @@ class create_topic_fixture : public redpanda_thread_fixture {
         client.stop().then([&client] { client.shutdown(); }).get();
     }
 
+    void test_create_non_replicated_topic(
```

Comment:
a hunch is that cluster/tests (as opposed to kafka/tests) might also have convenient fixtures for interacting with your cluster-level changes